### ISSUE TITLE
build failing on Windows with LDC v2.071.2 (mixin accessibility & deprecation of std.stream)

### ISSUE THREAD

**Issue by Stuey2** [Pure-D/workspace-d#50]
running install.bat gives the following output on Windows - Using the VS2015 x64 Native Tools command prompt.

Welcome to the workspace-d installation guide.
Make sure, you have dub and git installed.

LDC is required on your platform!

Which optional dependencies do you want to install?
[1] DCD - auto completion
[2] DScanner - code linting
[3] dfmt - code formatting
Enter a comma separated list of numbers
Selected [all]:
Cloning workspace-d into C:\Users\BlartyFob\AppData\Local\Temp\workspaced-install-636120400340230071
$ git clone -q https://github.com/Pure-D/workspace-d.git workspace-d
Checking out v2.7.2
$ git checkout -q v2.7.2
Compiling...
$ dub upgrade
Upgrading project in C:\Users\BlartyFob\AppData\Local\Temp\workspaced-install-636120400340230071\workspace-d
$ dub build --compiler=ldc2 --combined --build=release
Performing "release" build using ldc2 for x86.
workspace-d 2.7.2: building configuration "application"...
........\Roaming\dub\packages\dub-1.0.0\dub\source\dub\internal\sdlang\lexer.d(16): Deprecation: module std.stream is deprecated - It will be removed from Phobos in October 2016. If you still need it, go to https://github.com/DigitalMars/undeaD
source\app.d-mixin-222(222): Error: function workspaced.com.dub.start is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dub.restart is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dub.getInfo is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dub.listDependencies is not accessible from module app
source\app.d(270): Error: template instance workspaced.app.handleRequestMod!(dub) error instantiating
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.clientArgs is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.doClient is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.raw is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.execClient is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.rawExec is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.isPortRunning is not accessible from module app
source\app.d-mixin-222(222): Error: function workspaced.com.dcd.findOpen is not accessible from module app
source\app.d(272): Error: template instance workspaced.app.handleRequestMod!(dcd) error instantiating
source\app.d-mixin-222(222): Error: function workspaced.com.dscanner.raw is not accessible from module app
source\app.d(276): Error: template instance workspaced.app.handleRequestMod!(dscanner) error instantiating
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(481): Error: template painlessjson.painlessjson.defaultFromJSONImpl cannot deduce function from argument types !(dchar, SerializationOptions(true, false))(const(JSONValue)), candidates are:
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(324): painlessjson.painlessjson.defaultFromJSONImpl(T, SerializationOptions options)(in JSONValue json) if (is(T == JSONValue))
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(329): painlessjson.painlessjson.defaultFromJSONImpl(T, SerializationOptions options)(in JSONValue json) if (isIntegral!T)
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(334): painlessjson.painlessjson.defaultFromJSONImpl(T, SerializationOptions options)(in JSONValue json) if (isFloatingPoint!T)
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(342): painlessjson.painlessjson.defaultFromJSONImpl(T, SerializationOptions options)(in JSONValue json) if (is(T == string))
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(347): painlessjson.painlessjson.defaultFromJSONImpl(T, SerializationOptions options)(in JSONValue json) if (isBoolean!T)
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(481): ... (4 more, -v to show) ...
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(616): Error: template instance painlessjson.painlessjson.defaultFromJSON!(dchar, SerializationOptions(true, false)) error instantiating
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(358): instantiated from here: fromJSON!(dchar, SerializationOptions(true, false))
C:\Users\BlartyFob\LDC-master-2554-multilib\bin..\include\d\std\algorithm\iteration.d(581): instantiated from here: __lambda2!(const(JSONValue))
C:\Users\BlartyFob\LDC-master-2554-multilib\bin..\include\d\std\algorithm\iteration.d(480): instantiated from here: MapResult!(__lambda2, const(JSONValue)[])
........\Roaming\dub\packages\painlessjson-1.3.7\painlessjson\source\painlessjson\painlessjson.d(358): ... (3 instantiations, -v to show) ...
source\app.d-mixin-222(222): instantiated from here: fromJSON!(const(string), SerializationOptions(true, false))
source\app.d(278): instantiated from here: handleRequestMod!(dlangui)
........\Roaming\dub\packages\dub-1.0.0\dub\source\dub\compilers\utils.d(92): Deprecation: dub.compilers.buildsettings.array is not visible from module utils
ldc2 failed with exit code 1.
Error while compiling workspace-d.

... More info on ldc2 --version

LDC - the LLVM D compiler (f77d3f):
based on DMD v2.071.2 and LLVM 3.9.0git-fbbabf3
built with LDC - the LLVM D compiler (1.0.0)
Default target: x86_64-pc-windows-msvc
Host CPU: haswell
http://dlang.org - http://wiki.dlang.org/LDC

Registered Targets:
aarch64 - AArch64 (little endian)
aarch64_be - AArch64 (big endian)
amdgcn - AMD GCN GPUs
arm - ARM
arm64 - ARM64 (little endian)
armeb - ARM (big endian)
bpf - BPF (host endian)
bpfeb - BPF (big endian)
bpfel - BPF (little endian)
hexagon - Hexagon
mips - Mips
mips64 - Mips64 [experimental]
mips64el - Mips64el [experimental]
mipsel - Mipsel
msp430 - MSP430 [experimental]
nvptx - NVIDIA PTX 32-bit
nvptx64 - NVIDIA PTX 64-bit
ppc32 - PowerPC 32
ppc64 - PowerPC 64
ppc64le - PowerPC 64 LE
r600 - AMD GPUs HD2XXX-HD6XXX
sparc - Sparc
sparcel - Sparc LE
sparcv9 - Sparc V9
systemz - SystemZ
thumb - Thumb
thumbeb - Thumb (big endian)
x86 - 32-bit X86: Pentium-Pro and above
x86-64 - 64-bit X86: EM64T and AMD64
xcore - XCore

**Comment by WebFreak001** [comment]
pull and try again now, just forgot to bump the version because i have already fixed this.

Also the deprecation comes from dub (package), nothing I can do about it

**Comment by WebFreak001** [comment]
fixed with creation of tag v2.7.3 of commit ca3897d
